logging/logging.go

```diff
@@ -301,16 +302,15 @@ func (c *Client) Logger(logID string, opts ...LoggerOption) *Logger {
 	return l
 }
 
-type severityWriter struct {
+type protoEntryWriter struct {
 	l *Logger
-	s Severity
+	e *Entry // A prototype Entry object
 }
 
-func (w severityWriter) Write(p []byte) (n int, err error) {
-	w.l.Log(Entry{
-		Severity: w.s,
-		Payload:  string(p),
-	})
+func (w protoEntryWriter) Write(p []byte) (n int, err error) {
```

Comment:
nit: The `proto` prefix usually is typically used to mean `protobuf` elsewhere in the codebase. I think this should be renamed to reduce potential confusion. Maybe call it a `template` instead?

Reply:
Done.

Good idea. I also think of `proto` to mean proto message. `template` is better.